indy-besu/smart_contracts/contracts/cl/ClErrors.sol

```diff
@@ -68,6 +62,5 @@ error CredentialDefinitionNotFound(string id);
 /**
  * @notice Error that occurs when issuer DID of Schema or CredentialDefinition is not owned by sender.
  * @param sender Sender account address.
- * @param owner DID owner account address.
  */
-error SenderIsNotIssuerDidOwner(address sender, address owner);
+error UnauthorizedSender(address sender);
```

Comment:
We already have an "unauthorized" error in AuthError.sol, the names is very similar, could this lead to confusion in the future?